x/ibc/03-connection/client/cli/tx.go

```diff
@@ -282,6 +281,9 @@ func GetCmdHandshakeState(storeKey string, cdc *codec.Codec) *cobra.Command {
 		Short: "initiate connection handshake between two chains",
```

Comment:
ditto, should we split this command? open to suggestions

Reply:
I agree, I think we should.